### ISSUE TITLE
[PUBLIC TESTNET] fix faucet

### ISSUE THREAD

**Issue by verabehr** [VolumeFi/paloma#2070]
The faucet for public testnet fails when attempting to get tokens. https://faucet.palomaswap.com/
FE console shows a 400 error 
```
Failed to load resource: the server responded with a status of 400 (Bad Request) backend.faucet.palomaswap.com/claim:1 
```     

Backend service error:
```
Aug 27 20:26:50 testnet-linode faucet[579445]: error getting the balance information
Aug 27 20:26:50 testnet-linode faucet[579445]: Error: post failed: Post "http://23.92.27.95:26657": dial tcp 23.92.27.95:26657: connect: connection refused
```

Faucet setup details here https://www.notion.so/volumefi/Paloma-testnet-faucet-6dc1ac9b65e545e885d1958f02778b4c?pvs=4

**Comment by maharifu** [comment]
There are several issues here:

1. The palomad binary is listening on port 26657 on localhost only, so I changed the config of gofaucet.service to point to 127.0.0.1
2. There is an issue parsing the balance response (maybe it changed in some paloma upgrade?), so all requests fail.
3. The faucet private key is stored in the `os` keyring, but the gofaucet process has no code to decrypt it, so the send operation fails.

@verabehr @taariq I don't have write permissions to the [faucet repo](https://github.com/palomachain/faucet). Can you give me access to open a PR?
Also, since we don't have any code to unlock the keyring currently, I guess we were using the `test` keyring before and storing the key unencrypted. Should we just configure it this way again to move fast, since this is just the testnet?

**Comment by taariq** [comment]
@maharifu you are now admin.

**Comment by taariq** [comment]
@maharifu is this fixed with https://github.com/palomachain/faucet/pull/11#event-14103757172

**Comment by maharifu** [comment]
> @maharifu is this fixed with [palomachain/faucet#11 (comment)](https://github.com/palomachain/faucet/pull/11#event-14103757172)

That wasn't the only thing holding it back. I had to change the config as well. Anyway, we don't get an error anymore, but I don't get grain either, so I need to check it further.

**Comment by maharifu** [comment]
The testnet halted due to the v2.1.1 update. I updated all nodes and the faucet is now working again.
